src/land/UberArcanistStackSubmitQueueEngine.php

```diff
@@ -178,18 +178,39 @@ private function rebase($targetBranch, $ontoBranch, $verbose) {
         echo phutil_console_format(pht('Rebasing **%s** onto **%s**', $targetBranch, $ontoBranch) . "\n");
       }
       if (!$verbose) {
-        $this->runCommandSilently(array("echo", "y", "|", "git","rebase", pht("%s", $ontoBranch)));
+        try {
+          $this->runCommandSilently(array('echo', 'y', '|', 'git', 'rebase', pht('%s', $ontoBranch)),
+                                          $this->getUsesArcFlow());
+        } catch (Exception $e) {
+          if (!$this->getUsesArcFlow()) {
+            throw $e;
+          }
+          $this->writeInfo('ARC_REBASE',
+            pht('Unable to use standard rebase, trying alternative - rebase --onto %s %s^',  $ontoBranch, $targetBranch));
+          $repository_api->execxLocal('rebase --abort');
+          $repository_api->reloadWorkingCopy();
+          $this->runCommandSilently(array('echo', 'y', '|', 'git', 'rebase', '--onto', pht('%s', $ontoBranch), pht('%s^', $targetBranch)));
```

Comment:
I don't think `pht` call is required here.

Reply:
I'm following standard which was used in this file, see `181` line of old file